python/Framework.py

```diff
@@ -98,14 +92,12 @@ def __init__(self, options, verbose=True):
 
         self.configureFramework_()
 
+    @dep(before="create", performs="create", fallback=lambda self : self.process)
```

Comment:
what's the fallback for?

Reply:
It used to return the process if `create` had been called before... I'm not sure whether that's used anywhere (it would indeed be nicer not to need this, we could also make `process` a property that triggers `create` if not already done so, and use `_process` instead internally, which one do you prefer?)

Reply:
mmm, now this rings a (distant) bell, but this was not present before ?

Reply:
it used to start with `if self.__created: return self.process`, but indeed this feature seems not to be used, as far as I can see, so we can remove it, if you agree

Reply:
agreed then!